CHANGELOG.md

```diff
@@ -23,6 +23,7 @@
 - Fixes `customs_item.value` to ve a `number` instead of a `string`
 - Add support for `columns` and `additional_columns` on Report creation
 - Adds `address.verifyAddress()` and `Address.createAndVerify()` functions
+- Adds `Refund` class
```

Comment:
Can we instead add what functions of the Refund class were added? Simply stating `Refund` got added doesn't help users know what is now accessible.